### PR TITLE
Factor out meshopt into its own class

### LINKED ISSUE #10822: DracoCompression.Configuration no longer supports relative paths (regression)
Specifying a relative path to a local copy of draco decompression script files no longer works. It was working in BJS 4.20 but not working in BJS 5.0.0-alpha.37.

Discussed on forum here: https://forum.babylonjs.com/t/dracocompression-configuration-not-working-in-5-0-0-alpha-37/22828

Identified by @bghgary as  a regression from this PR: meshopt fixes and misc cleanup · BabylonJS/Babylon.js@95db4c8 

**Repro**
specify a relative path for DracoCompression like the following:
`
DracoCompression.Configuration = {
            decoder: {
              wasmUrl:  '/assets/draco/draco_wasm_wrapper_gltf.js',
              wasmBinaryUrl:  '/assets/draco/draco_decoder_gltf.wasm',
              fallbackUrl:  '/assets/draco/draco_decoder_gltf.js',
            }
        }
`
- Expected result: files are found and used
- Current result: Error message is generated: Failed to execute ‘importScripts’ on ‘WorkerGlobalScope’: The URL ‘/assets/draco/draco_wasm_wrapper_gltf.js’ is invalid.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 92.0.4515.131 

**Additional context**

### PR DESCRIPTION
**Pull request by bghgary**
Also allow relative paths in draco and meshopt configuration.

Fixes #10822

### CODE REVIEW

File: localDev/index.html [hunk 1]
```diff
@@ -89,11 +89,12 @@
                     wasmBinaryUrl: GetAbsoluteUrl("../dist/preview%20release/draco_decoder_gltf.wasm"),
                     fallbackUrl: GetAbsoluteUrl("../dist/preview%20release/draco_decoder_gltf.js")
                 };
+                BABYLON.MeshoptCompression.Configuration = {
+                    url: GetAbsoluteUrl("../dist/preview%20release/meshopt_decoder.js")
```
Comment (Popov72) [inline localDev/index.html:93]: Missing the `decoder` parent

File: sandbox/public/index-local.html [hunk 2]
```diff
@@ -45,11 +45,12 @@
                     wasmBinaryUrl: GetAbsoluteUrl("../../dist/preview%20release/draco_decoder_gltf.wasm"),
                     fallbackUrl: GetAbsoluteUrl("../../dist/preview%20release/draco_decoder_gltf.js")
                 };
+                BABYLON.MeshoptCompression.Configuration = {
+                    url: GetAbsoluteUrl("../../dist/preview%20release/meshopt_decoder.js")
```
Comment (Popov72) [inline sandbox/public/index-local.html:49]: Missing the `decoder` parent

**Review by Popov72 — APPROVED** [review]

**Review by RaananW — CHANGES_REQUESTED** [review]
Setting to request changes until comments are resolved

**Comment by RaananW** [comment]
You commited all of the dist folder as well. Was that intentional?

**Comment by bghgary** [comment]
Nope

**Review by RaananW — APPROVED** [review]

**Pull request merged**